raiden-dapp/src/services/raiden-service.ts

```diff
@@ -239,6 +239,15 @@ export default class RaidenService {
       this.store.commit('updateTokens', cache);
     }
   }
+
+  async transfer(token: string, target: string, amount: BigNumber) {
+    try {
+      await this.raiden.getAvailability(target);
```

Comment:
Can be done later, in another PR, but it's interesting to check `getAvailability` as soon as the address is input, and require/validate that the `available` is `true`. No need to cache the return of this function, as it's cached internally by the actions system, but be sure to only pass addresses which were already validated to it.

Reply:
Yes let's open an issue about this

Reply:
Opened the issue https://github.com/raiden-network/light-client/issues/218